modules/gitops-creating-a-new-client-using-keycloak.adoc

```diff
@@ -0,0 +1,45 @@
+[id="creating-a-new-client-in-keycloak_{context}"]
+= Configuring a new client in Keycloak
+
+Keycloak acts as an identity broker between Argo CD and OpenShift; as a result, you can log in to Argo CD using the OpenShift credentials.
+
+.Procedure
+
+. Dex is installed by default for all the Argo CD instances created by the operator. For Keycloak to work, first delete the dex configuration from the Argo CD CR. Remove the following and *Save* the CR:
```

Comment:
```suggestion
Dex is installed by default for all the Argo CD instances created by the Operator. For Keycloak to work, follow these steps:
. Delete the Dex configuration from the Argo CD CR by removing the following and saving the CR:
```
- The first sentence is not a step, and so shouldn't be in the procedure. Maybe use the first sentence as an intro to the procedure, as my edit shows.
- Per the OCP doc guidelines, `Operator` is always capitalized: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-capitalization
- `CR` should be spelled out on first use. See the "Spelled-out forms" section of the IBM SG.